packages/cli/openapi-ir-to-fern/src/__test__/__snapshots__/devrev.test.ts.snap

```diff
@@ -93,8 +93,10 @@ exports[`open api parser devrev docs 1`] = `
               "body": {
                 "properties": {
                   "file_name": {
+                    "default": undefined,
                     "docs": "The name of the file that's being uploaded.",
                     "type": "string",
+                    "validation": undefined,
```

Comment:
can we update the openapi-ir-to-fern to only add `default` and `validation` if they are non empty? `fern write-definition` shouldn't be writing out `undefined` values